### PR TITLE
[Logs] Add serverless review updates

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
This PR adds applicable updates from serverless [PR 174](https://github.com/elastic/staging-serverless-observability-docs/pull/174) to classic docs.

### CODE REVIEW

File: docs/en/observability/logs-filter.asciidoc [hunk 1]
```diff
@@ -92,20 +92,28 @@ POST logs-example-default/_bulk
 { "message": "2023-09-20T09:40:32.345Z INFO 192.168.1.106 User logout initiated." }
 ----
 
-From Log Explorer, add the following KQL query in the search bar to filter for logs within a timestamp range of September 14th and 15th with log levels of `WARN` or `ERROR`:
+For this example, let's look for logs with a `WARN` or `ERROR` log level that occurred on September 14th or 15th. From Log Explorer:
 
+1. Add the following KQL query in the search bar to filter for logs with log levels of `WARN` or `ERROR`:
```
Comment (colleenmcginnis) [inline docs/en/observability/logs-filter.asciidoc:97]:
```suggestion
. Add the following KQL query in the search bar to filter for logs with log levels of `WARN` or `ERROR`:
```

File: docs/en/observability/logs-filter.asciidoc [hunk 2]
```diff
@@ -92,20 +92,28 @@ POST logs-example-default/_bulk
...
 [source,text]
 ----
-@timestamp >= "2023-09-14T00:00:00" and @timestamp <= "2023-09-15T23:59:59" and log.level : "ERROR" or  log.level : "WARN"
+log.level: ("ERROR" or "WARN")
 ----
+2. Click the current time range, select **Absolute**, and set the **Start date** to `Sep 14, 2023 @ 00:00:00.000`.
++
+[role="screenshot"]
+image::images/logs-start-date.png[Set the start date for your time range, 50%]
+3. Click the end of the current time range, select **Absolute**, and set the **End date** to `Sep 15, 2023 @ 23:59:59.999`.
```
Comment (colleenmcginnis) [inline docs/en/observability/logs-filter.asciidoc:107]:
```suggestion
. Click the current time range, select **Absolute**, and set the **Start date** to `Sep 14, 2023 @ 00:00:00.000`.
+
[role="screenshot"]
image::images/logs-start-date.png[Set the start date for your time range, 50%]
. Click the end of the current time range, select **Absolute**, and set the **End date** to `Sep 15, 2023 @ 23:59:59.999`.
```

**Review by colleenmcginnis — DISMISSED** [review]
Don't forget to backport!

File: docs/en/observability/logs-filter.asciidoc (hunk 2 above)
Comment (mdbirnstiehl) [reply docs/en/observability/logs-filter.asciidoc:107]: Whoops, forgot how to do  Asciidoc.

**Review by colleenmcginnis — APPROVED** [review]
🍕

**Pull request merged**